esphomeyaml/cookbook/dual-r2-cover.rst

```diff
@@ -0,0 +1,113 @@
+Dual relay cover motor control
+==============================
+
+The following is a possible configuration file for common covers that use a motor with 2 inputs. Only one should be powered at a time (interlocking) to either move the cover up or down. For this the [Sonoff Dual R2](https://www.itead.cc/sonoff-dual.html) can be used which has two independent relays. Additionally this configuration allows the single button on the Sonoff to control the motion by cycling between: open->stop->down->stop->...
```

Comment:
Markdown links do not work in RST (`[Sonoff Dual R2](https://www.itead.cc/sonoff-dual.html)`), write

```rst
`Sonoff Dual R2 <https://www.itead.cc/sonoff-dual.html>`__ 
```
instead. See https://esphomelib.com/esphomeyaml/guides/contributing.html#contributing-to-esphomedocs

Reply:
Sorry my bad, force of habit. I'm mostly write md :)